f2/src/components/formik/datePicker/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { Form, Row, Col, Container } from 'react-bootstrap'
+import { Trans } from '@lingui/macro'
+import { Field } from 'formik'
+
+const DateEntry = ({ field, form, ...props }) => {
+  return (
+    <Col sm="auto" className="date-picker">
+      <Form.Group controlId={props.id}>
+        <Form.Label className="date-label">
+          {<Trans id="whenDidItStart.startDay" />}
+        </Form.Label>
+        <Form.Control
```

Comment:
Can we reuse our new Input component for all three input fields, date, month and year?

Reply:
Done, DateEntry component is reused for each Field.